docs/source/guide/pec-1-intro.myst

```diff
@@ -0,0 +1,142 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# How do I use PEC?
+
+Probabilistic Error Cancellation (PEC) is an error mitigation technique for improving the estimation
+of expectation values (see [What is the theory behind PEC?](pec-5-theory.myst)). To be applied, PEC requires the 
+full knowledge of the noise model. In particular, the user must know how to represent ideal gates as linear 
+combinations of noisy implementable gates. This knowledge is exploited by Mitiq
+to probabilistically remove the bias error from noisy expectation values.
+
++++
+
+The following code is compatible with the multiple frontends. The next code cell can be used for selecting one of them.
+
+```{code-cell} ipython3
+frontend = "qiskit"  # Supported: "cirq", "qiskit", "pyquil", "braket", "pennylane".
+```
+
+## Define the circuit of interest
+
++++
+
+We first define the circuit of interest. For simplicity, in this example we use
+a single-qubit randomized-benchmarking circuit whose ideal execution is equivalent to the
```

Comment:
Here "ideal" is used again without having been explicitly defined. I think it's a good idea to add definitions after the introduction.

"To apply PEC, you must know how to represent *ideal gates* as linear combinations of *noisy implementable gates*. By *ideal gates*, we mean noiseless unitary operations. By *noisy implementable gates* we mean..."